README.md

```diff
@@ -85,6 +97,12 @@ Define a global variable instructor and set it equal to your Squad Instructor's
 Your Answer:
 ```js
 // write code here
+var instructor = becky
+
```

Comment:
I think for this question, I was just looking for you to define two variables, one of them global versus local. Example:
```

// global variable
instructor = "Becky";
// local variable
var instructor = "Becky";
```

Additionally, because we haven't defined becky, we need to set var instructor = to a string, like `var instructor = "becky" `over `var instructor = becky`